### PR TITLE
Query: API review changes

### PR DESCRIPTION
**Pull request by smitpatel**
Part of #27588

### CODE REVIEW

**Comment by AndriySvyryd** [comment]
Not rc-1?

File: src/EFCore.Relational/Query/SetPropertyCalls.cs [hunk 1]
```diff
@@ -18,16 +18,20 @@ namespace Microsoft.EntityFrameworkCore.Query;
 ///     and <see href="https://aka.ms/efcore-docs-how-query-works">How EF Core queries work</see> for more information and examples.
 /// </remarks>
 /// <typeparam name="TSource">The type of source element on which ExecuteUpdate operation is being applied.</typeparam>
-public sealed class SetPropertyStatements<TSource>
+public sealed class SetPropertyCalls<TSource>
 {
+    private SetPropertyCalls()
```
Comment (bricelam) [inline src/EFCore.Relational/Query/SetPropertyCalls.cs:23]: Might be worth hiding ToString, Equals, and GetHashCode like we do on the model builder APIs

File: src/EFCore.Relational/Query/SqlExpressions/ColumnValueSetter.cs [hunk 2]
```diff
@@ -12,14 +12,14 @@ namespace Microsoft.EntityFrameworkCore.Query.SqlExpressions;
 ///         not used in application code.
 ///     </para>
 /// </summary>
-public class SetColumnValue
+public class ColumnValueSetter
```
Comment (AndriySvyryd) [inline src/EFCore.Relational/Query/SqlExpressions/ColumnValueSetter.cs:15]: Should this be in `Microsoft.EntityFrameworkCore.Query` namespace instead?

**Review by bricelam — APPROVED** [review]

**Review by AndriySvyryd — APPROVED** [review]

File: src/EFCore.Relational/Query/SqlExpressions/ColumnValueSetter.cs (hunk 2 above)
Comment (smitpatel) [reply src/EFCore.Relational/Query/SqlExpressions/ColumnValueSetter.cs:15]: Follows CaseWhenClause, it is part of Sql tree though not exactly expression.

**Comment by smitpatel** [comment]
note to self: retarget to rc1

**Pull request closed without merging**